### ISSUE TITLE
TypeError: Runner(): incompatible function arguments.

### ISSUE THREAD

**Issue by udara28** [Xilinx/Vitis-In-Depth-Tutorial#6]
I'm trying the 03_Basics/Module4/Step 7. When I execute the following command on ZCU104 board:
```
python3 app_mt.py
```

I get the following error:
```
Command line options:
 --image_dir :  images
 --threads   :  1
 --model     :  model_dir
Traceback (most recent call last):
  File "app_mt.py", line 183, in <module>
    main()
  File "app_mt.py", line 180, in main
    app(args.image_dir,args.threads,args.model)
  File "app_mt.py", line 105, in app
    all_dpu_runners.append(runner.Runner(model)[0])
TypeError: Runner(): incompatible function arguments. The following argument types are supported:
    1. (arg0: object, arg1: str) -> vart::Runner

Invoked with: 'model_dir'
```

What is causing this error? and How can I fix it?

Thank you.

Best Regards,
Udara

**Comment by rwarmstr** [comment]
Hi Udara, thanks for reporting the issue. I think we should hopefully have that resolved with #7, can you please update your repo source and let us know if you still see the issue?

Thanks,
Rob

**Comment by udara28** [comment]
@rwarmstr Thank you for your reply and Yes the problem is solved with the latest changes.
